PayPalCheckout/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 31
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        minSdkVersion 23
+        targetSdkVersion modules.androidTargetVersion
+        versionCode modules.sdkVersionCode
+        versionName modules.sdkVersionName
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            version = android.defaultConfig.versionName
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+}
+
+dependencies {
+    api project(':Core')
+
+    implementation 'com.paypal.checkout:android-sdk:0.5.1'
+
+    implementation deps.kotlinStdLib
+    implementation deps.androidxCoreKtx
+    implementation deps.androidxAppcompat
+
+    testImplementation deps.junit
+    androidTestImplementation deps.androidxJUnit
+    androidTestImplementation deps.androidxEspressoCore
+}
+
+project.ext.name = "checkout"
```

Comment:
I might have missed this - was this module name already discussed and agreed on? `com.paypal.android.checkout` dependency doesn't seem obvious to me that this is for checkout with PayPal vs checkout with Venmo, Card, etc.

Reply:
Agreed, `checkout` is too general